nixos/modules/services/misc/gitea.nix

```diff
@@ -947,7 +964,12 @@ in
       wantedBy = [ "timers.target" ];
       timerConfig.OnCalendar = cfg.dump.interval;
     };
+
+    services.postfix = mkIf cfg.mailerUseSendmail {
+      enable = true;
+    };
```

Comment:
sendmail is implementation-agnostic.  I don't think it makes sense for a module that uses sendmail to enable a specific implementation of it.

Reply:
This is necessary to get sendmail working correctly on the system.
https://github.com/NixOS/nixpkgs/blob/3e3afe5174c561dee0df6f2c2b2236990146329f/nixos/modules/services/mail/postfix.nix#L773-L780